RetailCoder.VBE/Inspections/Results/AssignedByValParameterInspectionResult.cs

```diff
@@ -25,9 +26,10 @@ public override IEnumerable<QuickFixBase> QuickFixes
         {
             get
             {
+                IAssignedByValParameterQuickFixDialogFactory factory = new AssignedByValParameterQuickFixDialogFactory();
```

Comment:
I went back and reviewed your earlier comments on this topic...I can see that they are consistent with the above comment - I think I just lost track of where to bring in the dependency once I started coding.  Soooo...I'll have another update coming soon.